class-jetpackbetaclicommand.php

```diff
@@ -51,13 +61,21 @@ private function validation_checks($args) {
 				return WP_CLI::error( __( 'Specify subcommand', 'jetpack' ) );
```

Comment:
It would be nice to say what the subcommands are. Since I got this error and I wasn't sure what to do next.